bot/cogs/fun.py

```diff
@@ -1,15 +1,16 @@
 """
 Set of bot commands designed for general leisure.
 """
-from random import choice, randint
+from random import randint
+from secrets import choice
```

Comment:
Do we really need two random number functions?

Our random quote picker really doesn't need to use the crypto number generator.